### PR TITLE
Add new scope for viewing preferences

### PR DESCRIPTION
**Pull request by iamtheluckyest**
Adds new scope for viewing preferences.

## Related Issues

- Links https://transcend.height.app/T-23447

### CODE REVIEW

File: src/scopes.ts [hunk 1]
```diff
@@ -617,6 +618,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
   },
+  [ScopeName.ViewPreferences]: {
+    title: 'View Preferences',
+    dependencies: [],
+    description: 'View preferences for users or organizations',
+    type: ScopeType.View,
+    products: [TranscendProduct.Admin],
```
Comment (iamtheluckyest) [inline src/scopes.ts:626]: I wasn't really sure what this should be if it's a scope that spans the AD. Should I have listed them all out?

**Comment by iamtheluckyest** [comment]
> do we need a scope for this? this is just the names of columns that are enabled or disabled right? doesnt seem to sensitive. feels like adding a scope would create more friction in configuration with little security benefit

@michaelfarrell76 I just didn't see an existing scope that was appropriate. Did I miss something?

**Comment by michaelfarrell76** [comment]
@iamtheluckyest i think you can just use `noScopes: true` on the endpoint

**Comment by iamtheluckyest** [comment]
> @iamtheluckyest i think you can just use `noScopes: true` on the endpoint

@michaelfarrell76 I'm not seeing any examples of that and it doesn't seem to be a property that exists on the endpoint.

**Comment by michaelfarrell76** [comment]
@iamtheluckyest apologies, its `noScope: true`

**Review by blake-transcend — CHANGES_REQUESTED** [review]
I don't think this PR is necessary, this endpoint should probably have no scope

**Comment by iamtheluckyest** [comment]
Got it! Thanks folks.

**Pull request closed without merging**